libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -2414,7 +2410,7 @@ TEST( Full_PKCS11_EC, AFQP_GetAttributeValueMultiThread )
         xGlobalTaskParams[ xTaskNumber ].pvTaskData = &xSessionHandle[ xTaskNumber ];
     }
 
-    prvProvisionEcTestCredentials( &xPrivateKey, &xCertificate, &xPublicKey );
+    prvFindObjectTest( &xPrivateKey, &xCertificate, &xPublicKey );
```

Comment:
Is there a way to fail with a valid message, if the credentials are not found.

Reply:
There are test assert messages in this helper function. It is tested first in AFQP_FindObjects, and then the code is reused in order to reuse objects between the tests. This was a miss in my earlier refactoring, as the goal is to only provision objects in the test cases that are testing that logic.